artifactory-users-report/Jenkinsfile

```diff
@@ -8,14 +8,20 @@ pipeline {
   options {
     // This pipeline takes 1-2 minutes max to execute
     timeout(time: 10, unit: 'MINUTES')
-    lock(resource: 'infra-reports-artifactory-users', inversePrecedence: true)
+    lock(resource: "artifactory-ldap-users-report-${env.BRANCH_NAME}", inversePrecedence: true)
```

Comment:
I'm not sure to understand why the BRANCH_NAME in in this resource name while it has been removed from the reportName line 2.

(same for each Jenkinsfile)

Reply:
Related to:

> Avoid locking PR builds when the main branch is running since no PR reports are published